--- elektra-0.8.19/tests/shell/CMakeLists.txt~	2016-11-22 21:59:21.000000000 +0100
+++ elektra-0.8.19/tests/shell/CMakeLists.txt	2017-11-23 08:45:00.306207515 +0100
@@ -82,7 +82,7 @@
 				set (KDB_COMMAND "${CMAKE_BINARY_DIR}/bin/kdb-full")
 			elseif (BUILD_STATIC)
 				set (KDB_COMMAND "${CMAKE_BINARY_DIR}/bin/kdb-static")
-			else (BUILD_SHARED)
+			elseif (BUILD_SHARED)
 				set (KDB_COMMAND "${CMAKE_BINARY_DIR}/bin/kdb")
 			else()
 				message(SEND_ERROR "no kdb tool found, please enable BUILD_FULL, BUILD_STATIC or BUILD_SHARED")
