--- elektra-0.8.23/src/bindings/io/uv/CMakeLists.txt~	2018-05-13 18:57:57.000000000 +0200
+++ elektra-0.8.23/src/bindings/io/uv/CMakeLists.txt	2021-03-04 23:06:03.297061786 +0100
@@ -54,39 +54,5 @@
 
 	install (FILES "${CMAKE_CURRENT_BINARY_DIR}/${IO_UV_LIBRARY}.pc" DESTINATION lib${LIB_SUFFIX}/${TARGET_PKGCONFIG_FOLDER})
 
-	# Build test
-	set (TESTEXENAME testio_${BINDING_VARIANT})
-
-	set (TEST_SOURCES $<TARGET_OBJECTS:cframework>)
-	add_headers (TEST_SOURCES)
-	file (GLOB IO_TEST_SRC_FILES "${CMAKE_SOURCE_DIR}/src/bindings/io/test/test*.c")
-	list (APPEND TEST_SOURCES ${IO_TEST_SRC_FILES})
-	list (APPEND TEST_SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/testio_${BINDING_VARIANT}.c")
-
-	if (BUILD_FULL OR BUILD_STATIC)
-		list (APPEND TEST_SOURCES $<TARGET_OBJECTS:OBJ_${IO_UV_LIBRARY}>) # add sources for elektra-io-uv for static and full builds
-	endif ()
-
-	add_executable (${TESTEXENAME} ${TEST_SOURCES})
-	add_dependencies (${TESTEXENAME} kdberrors_generated)
-
-	target_include_directories (${TESTEXENAME} PUBLIC "${CMAKE_SOURCE_DIR}/tests/cframework")
-
-	target_link_elektra (${TESTEXENAME} elektra-kdb elektra-plugin elektra-io ${IO_UV_LIBRARY} m)
-	if (BUILD_FULL OR BUILD_STATIC)
-		target_link_libraries (${TESTEXENAME} "${LIBUV_LIBRARIES};m")
-	endif ()
-
-	if (LIBUV_VERSION VERSION_LESS "1.0")
-		target_compile_definitions (${TESTEXENAME} PRIVATE "HAVE_LIBUV0")
-	else ()
-		target_compile_definitions (${TESTEXENAME} PRIVATE "HAVE_LIBUV1")
-	endif ()
-
-	add_test (NAME ${TESTEXENAME}
-		  COMMAND "${CMAKE_BINARY_DIR}/bin/${TESTEXENAME}" "${CMAKE_CURRENT_SOURCE_DIR}"
-		  WORKING_DIRECTORY "${WORKING_DIRECTORY}")
-	set_property (TEST ${TESTEXENAME} PROPERTY ENVIRONMENT "LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/lib")
-
 	add_subdirectory (example)
 endif ()
--- elektra-0.8.23/src/bindings/io/glib/CMakeLists.txt~	2018-05-13 18:57:57.000000000 +0200
+++ elektra-0.8.23/src/bindings/io/glib/CMakeLists.txt	2021-03-04 23:16:58.480231361 +0100
@@ -39,34 +39,5 @@
 
 	install (FILES "${CMAKE_CURRENT_BINARY_DIR}/${IO_VARIANT_LIBRARY}.pc" DESTINATION lib${LIB_SUFFIX}/${TARGET_PKGCONFIG_FOLDER})
 
-	# Build test
-	set (testexename testio_${BINDING_VARIANT})
-
-	set (TEST_SOURCES $<TARGET_OBJECTS:cframework>)
-	add_headers (TEST_SOURCES)
-	file (GLOB IO_TEST_SRC_FILES "${CMAKE_SOURCE_DIR}/src/bindings/io/test/test*.c")
-	list (APPEND TEST_SOURCES ${IO_TEST_SRC_FILES})
-	list (APPEND TEST_SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/testio_${BINDING_VARIANT}.c")
-
-	if (BUILD_FULL OR BUILD_STATIC) # add sources for elektra-io-glib for static and full builds
-		list (APPEND TEST_SOURCES $<TARGET_OBJECTS:OBJ_${IO_VARIANT_LIBRARY}>)
-	endif ()
-
-	add_executable (${testexename} ${TEST_SOURCES})
-	add_dependencies (${testexename} kdberrors_generated)
-
-	target_include_directories (${testexename} PUBLIC "${CMAKE_SOURCE_DIR}/tests/cframework")
-	target_include_directories (${testexename} PUBLIC SYSTEM ${GLIB_INCLUDE_DIRS})
-
-	target_link_elektra (${testexename} elektra-kdb elektra-plugin elektra-io ${IO_VARIANT_LIBRARY} m)
-	if (BUILD_FULL OR BUILD_STATIC)
-		target_link_libraries (${testexename} ${GLIB_LIBRARIES})
-	endif ()
-
-	add_test (NAME ${testexename}
-		  COMMAND "${CMAKE_BINARY_DIR}/bin/${testexename}" "${CMAKE_CURRENT_SOURCE_DIR}"
-		  WORKING_DIRECTORY "${WORKING_DIRECTORY}")
-	set_property (TEST ${testexename} PROPERTY ENVIRONMENT "LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/lib")
-
 	add_subdirectory (example)
 endif ()
--- elektra-0.8.23/CMakeLists.txt~	2018-05-13 18:57:57.000000000 +0200
+++ elektra-0.8.23/CMakeLists.txt	2021-03-04 23:21:10.222371932 +0100
@@ -85,7 +85,6 @@
 
 	add_subdirectory (src)
 	add_subdirectory (examples)
-	add_subdirectory (benchmarks)
 
 	if (BUILD_TESTING)
 		add_subdirectory (tests)
