diff -Nurb o/Makefile n/Makefile
--- o/Makefile	1970-10-30 02:52:28.000000000 +0100
+++ n/Makefile	2004-11-09 01:31:53.322186704 +0100
@@ -3,7 +3,9 @@
 # $LastChangedBy: aviram $
 
 NAME=elektra
-CC=gcc -O3 -g -Wcomment -Wformat -Wimplicit-int -Wimplicit-function-declaration -Wparentheses -Wreturn-type -Wunused -Wuninitialized
+CC = cc
+CFLAGS ?= -ggdb -O0
+AFLAGS += -Wall
 XMLINCLUDES=`xml2-config --cflags`
 XMLLIBS=`xml2-config --libs`
 
@@ -16,7 +18,7 @@
 
 
 .c.o:
-	${CC} -g -fpic -o $@ -c $<
+	$(CC) $(CFLAGS) $(AFLAGS)  -fpic -o $@ -c $<
 
 
 
@@ -36,11 +38,11 @@
 	for x in ${DIRS}; do (cd "$$x"; make $@); done
 
 libregistry.so: registrystub.o libkdb.so
-	${CC} -shared -fpic -o $@ libkdb.so registrystub.o
+	$(CC) $(CFLAGS) $(AFLAGS)  -shared -fpic -o $@ libkdb.so registrystub.o
 
 
 libkdb.so: localkdb.o key.o
-	${CC} -shared -fpic -o $@ localkdb.o key.o
+	$(CC) $(CFLAGS) $(AFLAGS)  -shared -fpic -o $@ localkdb.o key.o
 
 
 
@@ -51,25 +53,25 @@
 
 
 key.o: key.c kdb.h kdbprivate.h
-	${CC} -g -fpic -o $@ -c $<
+	$(CC) $(CFLAGS) $(AFLAGS)  -fpic -o $@ -c $<
 
 
 
 
 
 localkdb.o: localkdb.c kdb.h kdbprivate.h
-	${CC} -g -fpic -c $<
+	$(CC) $(CFLAGS) $(AFLAGS)  -fpic -c $<
 
 
 
 
 kdb.o: kdb.c kdb.h kdbprivate.h
-	${CC} -g ${XMLINCLUDES} -fpic -c $<
+	$(CC) $(CFLAGS) $(AFLAGS)  ${XMLINCLUDES} -fpic -c $<
 
 
 
 kdb: kdb.o libkdb.so
-	${CC} -g -L. ${XMLLIBS} -lkdb -o $@ $<
+	$(CC) $(CFLAGS) $(AFLAGS)  -L. ${XMLLIBS} -lkdb -o $@ $<
 	# ld -shared -lc -lkdb -static -lxml2 -lz -o $@ $<
 
 
