--- elektra-0.8.23/scripts/CMakeLists.txt.orig	2018-06-21 21:19:48.567341007 +0200
+++ elektra-0.8.23/scripts/CMakeLists.txt	2018-06-21 21:20:44.280673708 +0200
@@ -77,7 +77,7 @@
 	else ()
 
 		# This Folder is at least the valid folder for Debian
-		install (FILES kdb_zsh_completion DESTINATION /usr/share/zsh/vendor-completions RENAME _kdb)
+		install (FILES kdb_zsh_completion DESTINATION /usr/share/zsh/site-functions RENAME _kdb)
 	endif ()
 endif ()
 
