diff -Nurb b/kdb.h n/kdb.h
--- b/kdb.h	1970-10-09 14:30:09.000000000 +0100
+++ n/kdb.h	2004-11-09 19:57:51.226251920 +0100
@@ -285,8 +285,8 @@
 int kdbGetValueByParent(const char *parentName, const char *baseName, char *returned,
 	size_t maxSize);
 
-int kdbGetComment(const char *keyname, char *returned, size_t maxSize);
-size_t kdbSetComment(const char *keyname, const char *comment);
+// not implemented int kdbGetComment(const char *keyname, char *returned, size_t maxSize);
+// not implemented size_t kdbSetComment(const char *keyname, const char *comment);
 
 int kdbStatKey(Key *key);
 int kdbGetKey(Key *key);
diff -Nurb b/registrystub.c n/registrystub.c
--- b/registrystub.c	1970-08-22 22:34:52.000000000 +0100
+++ n/registrystub.c	2004-11-09 19:58:50.723207008 +0100
@@ -85,14 +85,16 @@
 }
 
 
-int registryGetComment(char *keyname, char *returned,size_t maxSize) {
+/* kdbGetComment is not implemented
+ * int registryGetComment(char *keyname, char *returned,size_t maxSize) {
 	return kdbGetComment(keyname, returned,maxSize);
-}
+} */
 
 
-size_t registrySetComment(char *keyname, char *comment) {
+/* kdbSetComment is not implemented
+ * size_t registrySetComment(char *keyname, char *comment) {
 	return kdbSetComment(keyname, comment);
-}
+} */
 
 int registryStatKey(Key *key) {
 	return kdbStatKey(key);
