--- elektra-0.7.0/doc/Makefile.am.orig	2008-10-16 20:53:31.000000000 +0200
+++ elektra-0.7.0/doc/Makefile.am	2013-03-26 20:58:28.370640680 +0100
@@ -198,8 +198,8 @@
 	list=`find $$api_d`; for p in $$list; do \
 	if test -f "$$p"; then d=; else d="$(srcdir)/"; fi; \
 	f="`echo $$p | sed -e 's|^.*/||'`"; \
-	  echo " $(develdocDATA_INSTALL) $$d$$p $(DESTDIR)$(apihtmldir)/$$f"; \
-	  $(develdocDATA_INSTALL) $$d$$p $(DESTDIR)$(apihtmldir)/$$f; \
+	  echo " $(install_sh_DATA) $$d$$p $(DESTDIR)$(apihtmldir)/$$f"; \
+	  $(install_sh_DATA) $$d$$p $(DESTDIR)$(apihtmldir)/$$f; \
 	done
 
 uninstall-develdocDATA:
--- elektra-0.7.2/src/bindings/cpp/Makefile.am.orig	2010-04-27 09:17:25.000000000 +0200
+++ elektra-0.7.2/src/bindings/cpp/Makefile.am	2013-03-27 20:00:49.167160035 +0100
@@ -12,7 +12,7 @@
 
 objects = kdb.o key.o keyset.o
 
-lib_LIBRARIES = libelektra-cpp.a
+ulib_LIBRARIES = libelektra-cpp.a
 libelektra_cpp_a_SOURCES = $(sources)
 libelektra_cpp_a_DEPENDENCIES =
 libelektra_cpp_a_CXXFLAGS = $(cxxflags)
@@ -20,7 +20,7 @@
 libelektra_cpp_a_LIBADD =
 BUILT_SOURCES =
 
-lib_LTLIBRARIES = libelektra-cpp.la
+ulib_LTLIBRARIES = libelektra-cpp.la
 libelektra_cpp_la_SOURCES = $(sources)
 libelektra_cpp_la_DEPENDENCIES =
 libelektra_cpp_la_CXXFLAGS = $(cxxflags)
