--- elektra-0.8.23/cmake/Modules/FindDiscount.cmake.orig	2018-06-21 21:21:05.117340135 +0200
+++ elektra-0.8.23/cmake/Modules/FindDiscount.cmake	2018-06-21 21:52:21.290652045 +0200
@@ -64,7 +64,7 @@
 	# ____________________________________________________________________________
 	# Actions taken when all components have been found
 
-	find_package_handle_standard_args (DISCOUNT DEFAULT_MSG DISCOUNT_LIBRARIES DISCOUNT_INCLUDES MARKDOWN_EXECUTABLE)
+	find_package_handle_standard_args (DISCOUNT DEFAULT_MSG DISCOUNT_LIBRARIES DISCOUNT_INCLUDES)
 
 	if (DISCOUNT_FOUND)
 
