--- elektra-0.9.14/scripts/completion/CMakeLists.txt.orig	2023-03-16 16:11:44.000000000 +0100
+++ elektra-0.9.14/scripts/completion/CMakeLists.txt	2023-08-25 19:51:06.727057400 +0200
@@ -65,7 +65,7 @@ if (INSTALL_SYSTEM_FILES)
 		# This Folder is at least the valid folder for Debian
 		install (
 			FILES kdb_zsh_completion
-			DESTINATION /usr/share/zsh/vendor-completions
+			DESTINATION /usr/share/zsh/site-functions
 			RENAME _kdb
 			COMPONENT elektra-bin)
 	endif ()
