--- elektra/doc/Makefile~	2006-03-02 12:41:01.000000000 +0200
+++ elektra/doc/Makefile	2006-03-02 12:41:05.000000000 +0200
@@ -84,13 +84,10 @@
 	[ -d ${DESTDIR}/usr/share/man/man3 ] || mkdir -p ${DESTDIR}/usr/share/man/man3
 	[ -d ${DESTDIR}/usr/share/man/man5 ] || mkdir -p ${DESTDIR}/usr/share/man/man5
 	[ -d ${DESTDIR}/usr/share/man/man7 ] || mkdir -p ${DESTDIR}/usr/share/man/man7
-	find ${APIDIR}/html | cpio -pdm ${DESTDIR}/usr/share/doc/elektra-devel/
-	for m in `ls *.1`; do gzip -c --best $$m > ${DESTDIR}/usr/share/man/man1/$$m.gz; done
-#	for m in `ls *.3`; do gzip -c --best $$m > ${DESTDIR}/usr/share/man/man3/$$m.gz; done
-	gzip ${APIDIR}/man/man3/kdb*.3 ${APIDIR}/man/man3/key*.3 ${APIDIR}/man/man3/ks*.3
-	cp ${APIDIR}/man/man3/kdb*.3.gz ${APIDIR}/man/man3/key*.3.gz ${APIDIR}/man/man3/ks*.3.gz ${DESTDIR}/usr/share/man/man3/
-	for m in `ls *.5`; do gzip -c --best $$m > ${DESTDIR}/usr/share/man/man5/$$m.gz; done
-	for m in `ls *.7`; do gzip -c --best $$m > ${DESTDIR}/usr/share/man/man7/$$m.gz; done
+	for m in *.1; do gzip -c --best $$m > ${DESTDIR}/usr/share/man/man1/$$m.gz; done
+	for m in ${APIDIR}/man/man3/{kdb,key,ks}*.3; do gzip -c --best $$m > ${DESTDIR}/usr/share/man/man3/$$(basename $$m).gz; done
+	for m in *.5; do gzip -c --best $$m > ${DESTDIR}/usr/share/man/man5/$$m.gz; done
+	for m in *.7; do gzip -c --best $$m > ${DESTDIR}/usr/share/man/man7/$$m.gz; done
 #	cat kdb.3.xml | perl -e '{while (<>) {m/<function>(\w*)<\/function>/; print $$1 . "\n" if ($$1); }}' | \
 #		grep kdb | sort -u | while read f; do ln -s kdb.3.gz ${DESTDIR}/usr/share/man/man3/$$f.3.gz; done
 #	cat key.3.xml | perl -e '{while (<>) {m/<function>(\w*)<\/function>/; print $$1 . "\n" if ($$1); }}' | \
