--- elektra-0.9.14/scripts/cmake/Modules/FindLibGit2.cmake.orig	2023-08-27 09:11:27.128839611 +0200
+++ elektra-0.9.14/scripts/cmake/Modules/FindLibGit2.cmake	2023-08-27 09:12:15.285245392 +0200
@@ -15,7 +15,7 @@ find_file (LibGit2_VERSION_HEADER NAMES
 if (NOT LibGit2_VERSION_HEADER STREQUAL LibGit2_VERSION_HEADER-NOTFOUND)
 	file (
 		STRINGS ${LibGit2_VERSION_HEADER} LibGit2_VERSION
-		REGEX "#define LIBGIT2_VERSION[ \t]+\".+\""
+		REGEX "#define[ \t]+LIBGIT2_VERSION[ \t]+\".+\""
 		LIMIT_COUNT 1)
 	string (REGEX REPLACE ".*\"(.+)\"" "\\1" LibGit2_VERSION ${LibGit2_VERSION})
 endif (NOT LibGit2_VERSION_HEADER STREQUAL LibGit2_VERSION_HEADER-NOTFOUND)
